app/experimenter/nimbus-ui/src/components/PreviewURL/index.tsx

```diff
@@ -24,19 +25,33 @@ const CopyableURL: React.FC<CopyableURLProps> = ({
   if (collection) {
     url = `${url}&optin_collection=${collection}`;
   }
-  return <code onClick={() => clipboard.writeText(url)}>{url}</code>;
+  return (
+    <code onClick={() => clipboard.writeText(url)} style={{ cursor: "copy" }}>
```

Comment:
Silly question (and probably we’ll outside the scope of this PR), but couldn’t we just use an `<a href={url}>` style link and make it open a new tab instead of requiring users to copy-paste to/from the clipboard?

Reply:
Would probably get something like `Security Error: Content at https://github.com/mozilla/experimenter/issues/5230 may not load or link to about:studies`.

Reply:
Yeah, unfortunately we can't link directly to about: pages. We should probably add a note in the code somewhere about that

Reply:
Added comment above where we define the url.